custom_components/tahoma/binary_sensor.py

```diff
@@ -73,6 +73,13 @@
         device_class=binary_sensor.DEVICE_CLASS_VIBRATION,
         is_on=lambda state: state == STATE_DETECTED,
     ),
+    # DomesticHotWaterProduction/WaterHeatingSystem
+    OverkizBinarySensorDescription(
+        key="io:OperatingModeCapabilitiesState",
+        name="Energy Demand Status",
+        device_class=binary_sensor.DEVICE_CLASS_HEAT,
+        is_on=lambda state: state["energyDemandStatus"] == 0,
```

Comment:
Hopefully, for the moment we retrieve only one value from the json. If we need to retrieve a second value, we will have to rethink the description key system.

Reply:
Same...